plugins/woocommerce/tests/php/includes/data-stores/class-wc-order-data-store-cpt-test.php

```diff
@@ -452,7 +453,7 @@ public function test_create_draft_order_doesnt_trigger_hook() {
 
 		add_action( 'woocommerce_new_order', $callback );
 
-		$draft_statuses = array( 'auto-draft', 'checkout-draft' );
+		$draft_statuses = array( OrderStatus::AUTO_DRAFT, \Automattic\WooCommerce\Blocks\Domain\Services\DraftOrders::STATUS );
```

Comment:
Not essential, but I personally find this a tad verbose and, actually, harder to read than the string literals that came before it. Can we import `DraftOrders`?

Reply:
Sure! Done ✅